modules/openid_connect/db/migrate/20250710133700_add_oidc_group_links.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+#-- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See COPYRIGHT and LICENSE files for more details.
+#++
+
+class AddOidcGroupLinks < ActiveRecord::Migration[8.0]
+  def change
+    create_table :oidc_group_links do |t|
+      t.belongs_to :group, null: false, index: true, foreign_key: { to_table: :users, on_delete: :cascade }
+
+      t.belongs_to :auth_provider, null: false, index: false, foreign_key: { on_delete: :cascade }
+      t.string :oidc_name, null: false
```

Comment:
I suggest the following renaming. `oidc_name`  seems to be not specific enough. It can be confused with OIDC provider name, can't it?
WDYT?
```suggestion
      t.string :oidc_group_name, null: false
```

Reply:
That confusion didn't occur to me until you brought it up :sweat_smile: 

I have no strong objections against the renaming, except that it is longer than the original and makes lines longer and will probably result in new style violations afterwards xD

Did you find the name confusing yourself, when you encountered it, or was that remark out of a matter of caution? Maybe in other words: Would you rate it as a :green_circle: or a :yellow_circle: ?

Reply:
The name confused me at first and made me think what it is. And then I relized most likely it means group name on OIDC side. So, I rate it as   🟡

Reply:
changed to `oidc_group_name`